gooddata-api-client/LICENSE.txt

```diff
@@ -0,0 +1 @@
+../OSS LICENSES/LICENSE (gooddata-api-client).txt
```

Comment:
It looks like invalid symlink. I do not see `OSS LICENSES/LICENSE (gooddata-api-client).txt` being part of PR.